### ISSUE TITLE
Label issue in an event published mail.

### ISSUE THREAD

**Issue by krinaydhanani** [wpeventmanager/wp-event-manager#590]
Label issue in an event published mail.

Event name: - 

![image](https://user-images.githubusercontent.com/35419531/96248502-83f3c480-0f60-11eb-84e0-a38174ed4964.png)

What I get in the mail: - 

![image](https://user-images.githubusercontent.com/35419531/96248376-5575e980-0f60-11eb-8069-8051b344545a.png)

**Comment by wpem-kishan** [comment]
This is issue form email addon. We have fixed already.

**Comment by krinaydhanani** [comment]
Yes, this issue is from the email addon side so I will move this issue there.
